src/app/page.tsx

```diff
@@ -3,9 +3,16 @@ import styles from './page.module.css';
 
 export default function Home() {
   return (
-    <main className={styles.main}>
-      <h1>Welcome to Deer Tracker!</h1>
-      <h3>Powered by AI & Machine Learning</h3>
+    <main className={styles.mainpageContainer}>
+      <div className={styles.headerconatiner}>
+        <header className={styles.h1}>Welcome to Deer Tracker!</header>
+      </div>
+      <div className={styles.lgnbttncontainer}>
+        <button className={styles.loginbtton}>login</button>
+      </div>
+      <div className={styles.signbttncontainer}>
+        <button className={styles.signupbtton}>sign up</button>
+      </div>
```

Comment:
Move it to its own component called "Inactive Button" and capitalize the S in Sign up